tutorials/time-freq/plot_ssvep.py

```diff
@@ -0,0 +1,767 @@
+                  raw.info['line_freq'])
+raw.notch_filter(notch, filter_length='auto', phase='zero', verbose=False)
+
+# Apply bandpass filter
+hp = .1
+lp = 250.
+raw.filter(hp, lp, fir_design='firwin', verbose=False)
+
+# Construct epochs
+event_id = {
+    '12hz': 255,
+    '15hz': 155
+}
+events, _ = mne.events_from_annotations(raw, verbose=False)
+raw.info["events"] = events
+tmin, tmax = -1., 20.  # in s
+baseline = None
+epochs = mne.Epochs(
+    raw, events=events,
+    event_id=[event_id['12hz'], event_id['15hz']], tmin=tmin,
+    tmax=tmax, baseline=baseline, verbose=False)
+
+###############################################################################
+# Frequency analysis
+# ------------------
+# Now we compute the frequency spectrum of the EEG data.
+# You will already see the peaks at the stimulation frequencies and some of
+# their harmonics, without any further processing.
+#
+# The 'classical' PSD plot will be compared to a plot of the SNR spectrum.
+# SNR will be computed as a ratio of the power in a given frequency bin
+# to the average power in its neighboring bins.
+# This procedure has two advantages over using the raw PSD:
+#
+# * it normalizes the spectrum and accounts for 1/f power decay.
+#
+# * power modulations which are not very narrow band will disappear.
+#
+# Calculate power spectral density (PSD)
+# ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+# We use Welch's method for frequency decomposition, since it is really fast.
+# We chose a frequency resolution of 0.1 hz.
+# You could compare it with, e.g., multitaper to get an impression of the
+# influence on SNR. All the other methods implemented in MNE can be used as
+# well.
+#
+
+tmin = 0.
+tmax = 20.
+fmin = 1.
+fmax = 90.
+sf = epochs.info['sfreq']
+
+psds, freqs = mne.time_frequency.psd_welch(
+    epochs,
+    n_fft=int(sf * 10), n_overlap=int(sf * .5), n_per_seg=None,
+    tmin=tmin, tmax=tmax,
+    fmin=fmin, fmax=fmax, verbose=False)
+
+
+###############################################################################
+# Calculate signal to noise ratio (SNR)
+# ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+#
+# SNR - as we define it here - is a measure of relative power:
+# it's the ratio of power in a given frequency bin - the 'signal' -
+# to a 'noise' baseline - the average power in the surrounding frequency bins.
+#
+# Hence, we need to set some parameters for this baseline - how many
+# neighboring bins should be taken for this computation, and do we want to skip
+# the direct neighbors (this can make sense if the stimulation frequency is not
+# super constant, or frequency bands are very narrow).
+#
+# The function below does what we want.
+#
+
+def snr_spectrum(psd, noise_n_neighborfreqs=1, noise_skip_neighborfreqs=1):
+    """
+    Parameters
+    ----------
+    psd : ndarray
+        containing psd values as spit out by mne functions. must be 2d or 3d
+        with frequencies in the last dimension
+    noise_n_neighborfreqs - int
```

Comment:
```suggestion
    noise_n_neighborfreqs : int
```
to be fixed everywhere in docstring